src/processTargets/processTargets.ts

```diff
@@ -198,6 +208,10 @@ function processPrimitiveTarget(
     ...context.actionFinalStages,
   ];
 
+  if (!isRangeTarget) {
+    modifierStages.push(new ModifyUnTypedEmptyToTokenStage());
+  }
```

Comment:
How come you skip this on range targets?

Reply:
Because we run this on both the primaries before the range is constructed. I don't think I want `take air past this` or `this slice air` to expand to token?

Reply:
Huh.  This one feels a bit inconsistent, but I guess I agree.  Are there tests for this case?

Reply:
There is now :D